packages/astro/src/core/request.ts

```diff
@@ -1,15 +1,22 @@
 import type { IncomingHttpHeaders } from 'node:http';
+import { AstroError, AstroErrorData } from './errors/index.js';
 import type { Logger } from './logger/core.js';
 
 type HeaderType = Headers | Record<string, any> | IncomingHttpHeaders;
-type RequestBody = ArrayBuffer | Blob | ReadableStream | URLSearchParams | FormData;
+type RequestBody =
+	| ArrayBuffer
+	| Blob
+	| ReadableStream
+	| URLSearchParams
+	| FormData
+	| ReadableStream<Uint8Array>;
```

Comment:
This is type provided by `Request.body`

Reply:
Could we use `Request['body']` instead?

Reply:
Sorry @florian-lefebvre, I completely missed you comment. I'll follow up with another PR